l10n/en/firefox/installer-help.ftl

```diff
@@ -0,0 +1,31 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/installer-help/
+
+## Required
+installer-help-page-title = Your download was interrupted
+installer-help-main-heading = Hmm… your download was interrupted
+installer-help-main-tagline = That happens sometimes. Let’s try something different.
+installer-help-action-channel = This download should work better for you.
+installer-help-action-choose = Choose a download below.
+
+##
+installer-help-firefox-release-title = { -brand-name-firefox-browser }
+
+# Variables:
+#   $trackers (number) - number of trackers blocked by Firefox
```

Comment:
Should we give an approximate number here? Like "currently 2000" or "will be thousands".

Reply:
Good suggestion 👍